features/step_definitions/cookie_steps.rb

```diff
@@ -0,0 +1,10 @@
+Then /^I should receive a "([a-z0-9_]+)" cookie which is "([a-zA-Z]+)"$/ do |cookie_name, cookie_property|
+  headers = @response.headers[:set_cookie].select do |h|
```

Comment:
Can you use `response.cookies` here?

https://github.com/rest-client/rest-client#cookies

Might remove some of the funky regular expression stuff.